L/libRadtran/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libRadtran"
+version = v"2.0.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.libradtran.org/download/libRadtran-$(version).tar.gz", "eb840e00f2b59648e77775df83d8ae2337880cec155d145228cd65365e3c816f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libRadtran-2.0.4/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+touch lib/libRadtran.so
+make clean
+make
+make install
+mv lib/libRadtran.so ${prefix}/lib/libRadtran.${dlext}
+install_license COPYING
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions([
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; )
+])
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("make_angresfunc", :make_angresfunc),
+    ExecutableProduct("ssradar", :ssradar),
+    ExecutableProduct("uvspec_mpi", :uvspec_mpi),
+    ExecutableProduct("read_Stamnes_tab", :read_Stamnes_tab),
+    ExecutableProduct("zenith", :zenith),
+    ExecutableProduct("noon", :noon),
+    ExecutableProduct("sza2time", :sza2time),
+    ExecutableProduct("angres", :angres),
+    ExecutableProduct("uvspec", :uvspec),
+    ExecutableProduct("vac2air", :vac2air),
+    ExecutableProduct("vrs_ocean_albedo", :vrs_ocean_albedo),
+    ExecutableProduct("make_slitfunction", :make_slitfunction),
+    ExecutableProduct("pmom", :pmom),
+    ExecutableProduct("phase", :phase),
+    ExecutableProduct("sofi", :sofi),
+    ExecutableProduct("conv", :conv),
+    ExecutableProduct("mie", :mie),
+    ExecutableProduct("integrate", :integrate),
+    ExecutableProduct("snowalbedo", :snowalbedo),
+    ExecutableProduct("spline", :spline),
+    ExecutableProduct("cldprp", :cldprp),
+    ExecutableProduct("plkavg", :plkavg),
+    ExecutableProduct("time2sza", :time2sza),
+    ExecutableProduct("uvspecfunction", :uvspecfunction),
+    LibraryProduct("libRadtran", :libRadtran),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
```

Comment:
There are [250 warnings](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=22148&view=results) like this
```
┌ Warning: To ensure that the correct version of libgfortran is found at runtime, add the following entry to the list of dependencies of this builder
│ 
│     Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))
└ @ BinaryBuilder.Auditor /depot/packages/BinaryBuilder/1dtJX/src/auditor/compiler_abi.jl:4
```